python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -899,6 +899,111 @@ interface {{ ethernet_interface.name }}
       priority {{ ethernet_interface.sync_e.priority }}
 {%         endif %}
 {%     endif %}
+{%     if ethernet_interface.switchport.tap.native_vlan is arista.avd.defined %}
```

Comment:
To avoid all interfaces having to be checked for all these `ifs` I think we should enclose all of it into an outer `if ethernet_interface.switchport.tap is arista.avd.defined` and the same for tool.

Reply:
Fixed